client/src/app/site/pages/meetings/pages/projectors/modules/projector-detail/components/projector-detail/projector-detail.component.html

```diff
@@ -171,7 +171,7 @@ <h2>{{ projector.name }}</h2>
                     <hr />
                 </div>
 
-                <div>
+                <div class="spacer-bottom-60">
```

Comment:
Seems to be the wrong container for the spacer. This one creates a second scrollbar on smaller screens.
Tested in brave (chromium based).